src/jupyterlabcontroller/models/v1/lab.py

```diff
@@ -60,14 +61,12 @@ class LabStatus(Enum):
     FAILED = "failed"
 
     @classmethod
-    def from_phase(cls, phase: str) -> LabStatus:
+    def from_phase(cls, phase: PodPhase) -> LabStatus:
```

Comment:
I have a bad feeling we're also going to need to consider Reason here.  Lifecycle handling is such a mess.